### PR TITLE
Make 3 run-pass tests works on android (aarch64 and x86)

### PR DESCRIPTION
**Pull request by malbarbo**

### CODE REVIEW

File: src/test/run-pass/signal-alternate-stack-cleanup.rs [hunk 1]
```diff
@@ -31,11 +31,23 @@ extern fn send_signal() {
...
+
 fn main() {
     unsafe {
         // Install signal hander that runs on alternate signal stack.
         let mut action: sigaction = std::mem::zeroed();
-        action.sa_flags = SA_SIGINFO | SA_ONSTACK;
+        // workaround to build in android/aarch64 and android/x86_64
+        // the intention is:
+        // action.sa_flags = SA_ONSTACK | SA_SIGINFO;
+        // See https://github.com/rust-lang/libc/pull/511
```
Comment (malbarbo) [inline src/test/run-pass/signal-alternate-stack-cleanup.rs]: It this is too much, we can ignore this test in the affected targets.

Comment (kennytm) [reply src/test/run-pass/signal-alternate-stack-cleanup.rs]: Or you could write this as 

```rust
action.sa_flags = (SA_ONSTACK | SA_SIGINFO) as _;
```

Comment (malbarbo) [reply src/test/run-pass/signal-alternate-stack-cleanup.rs]: Oh, this is embarrassing... Fixed it.

**Pull request merged**